### PR TITLE
feat: port to Panacea

### PR DESCRIPTION
**Pull request by youngjoon-lee**
### Changes
- Changed an app name from `Cosmos` to `MediBloc`.
- Changed a coin type in the HD path from `118` to `371`.
- Changed a default Bech32 prefix from `cosmos` to `panacea`
- Changed app icons.
- Changed hardcoded values in unit tests.
   - Also, changed app screenshots for zemu tests

### Release plan
- Will be released as `v2.34.2-panacea` because this PR is a fork of `cosmos/ledger-cosmos@v2.34.2`.

### CODE REVIEW

File: docs/TXSPEC.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ Transactions passed to the Ledger device will be in the following format. The Le
 ```json
 {
   "account_number": "123",
-  "chain_id": "cosmoshub-4",
+  "chain_id": "panacea-3",
   "fee": {
-    "amount": [{"amount": "4000", "denom": "uatom"}, ...],
+    "amount": [{"amount": "4000", "denom": "umed"}, ...],
     "gas": "40000"
```
Comment (gyuguen) [inline docs/TXSPEC.md:32]: It seems that our chain has set a fee of 5umed and 200000 gas.

File: tests/json_parser.cpp [hunk 2]
```diff
@@ -365,7 +365,7 @@ namespace {
 
     TEST(TxValidationTest, ObjectGetValueCorrectFormat) {
         auto transaction =
-                R"({"account_number":"0","chain_id":"test-chain-1","fee":{"amount":[{"amount":"5","denom":"photon"}],"gas":"10000"},"memo":"testmemo","msgs":[{"inputs":[{"address":"cosmosaccaddr1d9h8qat5e4ehc5","coins":[{"amount":"10","denom":"atom"}]}],"outputs":[{"address":"cosmosaccaddr1da6hgur4wse3jx32","coins":[{"amount":"10","denom":"atom"}]}]}],"sequence":"1"})";
+                R"({"account_number":"0","chain_id":"test-chain-1","fee":{"amount":[{"amount":"5","denom":"photon"}],"gas":"10000"},"memo":"testmemo","msgs":[{"inputs":[{"address":"panaceaaccaddr1d9h8qat5e4ehc5","coins":[{"amount":"10","denom":"atom"}]}],"outputs":[{"address":"panaceaaccaddr1da6hgur4wse3jx32","coins":[{"amount":"10","denom":"atom"}]}]}],"sequence":"1"})";
```
Comment (gyuguen) [inline tests/json_parser.cpp:368]: It seems that you need to change the atom to med here.

File: docs/TXSPEC.md (hunk 1 above)
Comment (youngjoon-lee) [reply docs/TXSPEC.md:32]: This document is only for an example. The setting of 5umed and 200000gas is configured in each network. So, this ledger implementation doesn't need to care about those settings.

File: tests/json_parser.cpp (hunk 2 above)
Comment (youngjoon-lee) [reply tests/json_parser.cpp:368]: Actually, it doesn't matter when considering the way how this test works, but I will change it for clarity.

File: tests/testcases/manual.json [hunk 3]
```diff
@@ -595,12 +595,12 @@
     "name": "delegationExpert",
     "tx": {
       "account_number": "6571",
-      "chain_id": "cosmoshub-4",
+      "chain_id": "panacea-3",
       "fee": {
         "amount": [
           {
             "amount": "5000",
```
Comment (gyuguen) [inline tests/testcases/manual.json:602]: So is this
```suggestion
            "amount": "5",
```

File: docs/TXSPEC.md (hunk 1 above)
Comment (youngjoon-lee) [reply docs/TXSPEC.md:32]: Specifically, the ledger app always uses configurations that are specified by `--fees`, `--gas` and `--gas-price`.

File: tests/testcases/manual.json (hunk 3 above)
Comment (youngjoon-lee) [reply tests/testcases/manual.json:602]: We don't need to change this value.
https://github.com/medibloc/ledger-panacea/pull/1#discussion_r853955886

Comment (youngjoon-lee) [reply tests/testcases/manual.json:602]: Actually, we also don't need to change the chain ID from cosmoshub-4 to panacea-3.

File: tests/tx_parse.cpp [hunk 4]
```diff
@@ -141,7 +141,7 @@ namespace {
     }
 
     TEST(TxParse, Tx_Page_Count) {
-        auto transaction = R"({"account_number":"0","chain_id":"test-chain-1","fee":{"amount":[{"amount":"5","denom":"photon"}],"gas":"10000"},"memo":"testmemo","msgs":[{"inputs":[{"address":"cosmosaccaddr1d9h8qat5e4ehc5","coins":[{"amount":"10","denom":"atom"}]}],"outputs":[{"address":"cosmosaccaddr1da6hgur4wse3jx32","coins":[{"amount":"10","denom":"atom"}]}]}],"sequence":"1"})";
+        auto transaction = R"({"account_number":"0","chain_id":"test-chain-1","fee":{"amount":[{"amount":"5","denom":"photon"}],"gas":"10000"},"memo":"testmemo","msgs":[{"inputs":[{"address":"panaceaaccaddr1d9h8qat5e4ehc5","coins":[{"amount":"10","denom":"atom"}]}],"outputs":[{"address":"panaceaaccaddr1da6hgur4wse3jx32","coins":[{"amount":"10","denom":"atom"}]}]}],"sequence":"1"})";
```
Comment (gyuguen) [inline tests/tx_parse.cpp:144]: It is also marked as an atom here, is there any problem?

File: docs/TXSPEC.md (hunk 1 above)
Comment (gyuguen) [reply docs/TXSPEC.md:32]: I got it. Thank you

**Review by gyuguen — APPROVED** [review]
Thank you for your hard work.

**Comment by youngjoon-lee** [comment]
@gyuguen @hansol-medi 
Changed the app name from `Panacea` to `MediBloc`: [91bf466](https://github.com/medibloc/ledger-panacea/pull/1/commits/91bf46612a021d10ab7f1eaf70bad2425c91e988)
Changed app icons: [71db396](https://github.com/medibloc/ledger-panacea/pull/1/commits/71db3962df0fceaa9557aaa1decde759012ba256)

**Review by 0xHansLee — APPROVED** [review]
Thank you for your efforts!! 
We have our ledger app 🙌

**Pull request merged**